languages/readme.txt

```diff
@@ -0,0 +1,12 @@
+== For Translators ==
+
+Read more on how to translate .PO file https://developer.wordpress.org/plugins/internationalization/localization/#translate-po-file
+
+Some general translation tips https://developer.wordpress.org/plugins/internationalization/localization/#tips-for-good-translations
+
+If you have created your own translation or have an update of an existing one, please send it to dasha@dashaluna.com or create a pull request on github repository (github.com/dashaluna/hm-content-toc ), so I can bundle it into the next release of the plugin.
+
+Please only send .PO file, the corresponding .MO file will be generated by me.
+
+Thank you!
+Dasha
```

Comment:
@johnbillion do I need to add this text to the main `readme` file for the plugin?